test/services/test_connection_manager.py

```diff
@@ -1,7 +1,6 @@
 import unittest
 from unittest.mock import MagicMock
 from qiskit import IBMQ
-from qiskit import QuantumCircuit, BasicAer
 
 from src.services.connection_manager import account_initialization_manager, ConnectionManager
```

Comment:
Method `ConnectionManager` is not used.

```suggestion
from src.services.connection_manager import account_initialization_manager
```